cluster-operator/src/main/java/io/strimzi/operator/cluster/model/TopicOperator.java

```diff
@@ -210,6 +210,11 @@ public static String secretName(String cluster) {
         return cluster + io.strimzi.operator.cluster.model.TopicOperator.CERTS_SUFFIX;
     }
 
+    @Override
+    public String getGcLoggingOptions() {
```

Comment:
Again, does this override serve any purpose?

Reply:
In the AbstractModel  it returns `DEFAULT_KAFKA_GC_LOGGING`, in the EO `DEFAULT_STRIMZI_GC_LOGGING`. Should I add String parameter to `getGcLoggingOptions()`?

Reply:
Ah, OK, that's fine.